test/powershell/Language/Scripting/NativeExecution/NativeCommandArguments.Tests.ps1

```diff
@@ -287,97 +287,3 @@ foreach ( $argumentListValue in "Standard","Legacy","Windows" ) {
         }
     }
 }
-Describe 'PSPath to native commands' -tags "CI" {
```

Comment:
i suppose we should have a negative test here to ensure that we've removed the behavior we wanted to remove. Are there other validations that we should add?

Reply:
That's a good point, I'll add a test so that pspath is literal to a native command